src/index.js

```diff
@@ -294,6 +301,34 @@ const userEvent = {
     element.addEventListener("blur", fireChangeEvent);
   },
 
+  upload(element, fileOrFiles, init) {
+    if (element.disabled) return;
+    const focusedElement = element.ownerDocument.activeElement;
+
+    let files;
```

Comment:
This `files` variable depends on input `multiple` attribute. If it's true `fileOrFiles` arg assuming as array otherwise as singlefile. At first it relied on `Array.isArray` check but then I thought about passing test cases with multiple files and input without attribute `multiple`.